pkg/pointerdb/allocation.go

```diff
@@ -103,7 +103,7 @@ func (allocation *AllocationSigner) OrderLimit(ctx context.Context, parameters O
 	}
 
 	// convert bwExpiration from days to seconds
-	orderExpiration, err := ptypes.TimestampProto(time.Unix(int64(allocation.bwExpiration*86400), 0))
+	orderExpiration, err := ptypes.TimestampProto(time.Now().Add(time.Duration(allocation.bwExpiration*24) * time.Hour))
```

Comment:
`bwExpiration` probably should be `time.Duration` in the first place, but we can fix that later.

Reply:
let fix this later